upf/upf.h

```diff
@@ -519,21 +519,117 @@ typedef enum
   UPF_TIMERS_MISSED = 8,
   UPF_IPFIX_RECORDS_SENT = 9,
   UPF_IPFIX_MESSAGES_SENT = 10,
-  UPF_N_COUNTERS = 11,
+  // total of PFCP messages received - corrupted
+  UPF_PFCP_RECEIVED_CORRUPTED = 11,
+  // PFCP messages received
+  // _ERROR counter should always follow _OK counter
+  UPF_PFCP_RX_HB_REQUEST_OK = 12,
+  UPF_PFCP_RX_HB_REQUEST_ERROR = 13,
+  UPF_PFCP_RX_HB_RESPONSE_OK = 14,
+  UPF_PFCP_RX_HB_RESPONSE_ERROR = 15,
```

Comment:
what I meant with RX is that each counter may have this prefix to clarify that counter is for received message. So either all, either none

Reply:
huh thought that HB could be sent 2 ways, my bad
Id go without those prefixes